Andrey_Popok_3.5.py

```diff
@@ -0,0 +1,12 @@
+s = input("Enter the list of numbers into a single line divided by blanks: ")
+li1 = s.split(" ")
+list_num = []
+print("Your input after split:",li1)
+for i in range(len(li1)):
+    if li1[i].isdigit:
+        list_num.append(int(li1[i]))
+print("Your input after append:",list_num)
+sum = 0
```

Comment:
Don't forget thet **sum** is build-in function, and sometimes similar decision can be real headache